openlibrary/conftest.py

```diff
@@ -12,10 +11,6 @@
 from openlibrary.i18n import gettext
 from openlibrary.core import helpers
 
```

Comment:
```suggestion

from openlibrary.mocks.mock_infobase import mock_site  # noqa: F401
from openlibrary.mocks.mock_memcache import mock_memcache  # noqa: F401
```
Let’s use these linter directives to tell flake8 that we need these imports.

Reply:
Okay cool!